App/consts/globalProps.js

```diff
@@ -0,0 +1,11 @@
+export const customTextInputProps = {
```

Comment:
In some sence this naming is wrong. Just use something like appFontStyle. And that's more style then props